src/main/resources/org/csanchez/jenkins/plugins/kubernetes/ContainerTemplate/help-image.html

```diff
@@ -1,3 +1,3 @@
 Container image (repository, name, and tag) for a Jenkins inbound agent.
 Connection details will be passed as environment variables as per
-<a href="https://github.com/jenkinsci/docker-inbound-agent/#running-this-container">documentation</a>.
+<a href="https://github.com/jenkinsci/docker-agent/blob/master/README_inbound-agent.md#running-this-container">documentation</a>.
```

Comment:
Beware that this link can be invalidated. The corresponding permalink
```suggestion
<a href="https://github.com/jenkinsci/docker-agent/blob/3c9c39ed45279f7296a056f34c0c8f56e70345a9/README_inbound-agent.md#running-this-container">documentation</a>.
```
is safer to include in a (non-SaaS) product, though it would fail to keep up with useful edits.

Reply:
Updated.